x-pack/plugins/security_solution/public/resolver/view/clickthrough.test.tsx

```diff
@@ -208,6 +211,228 @@ describe('Resolver, when analyzing a tree that has no ancestors and 2 children',
   });
 });
 
+describe('Resolver, when using a generated tree with 20 generations, 4 children per child, and 10 ancestors', () => {
+  beforeEach(async () => {
+    const { metadata: dataAccessLayerMetadata, dataAccessLayer } = generateTreeWithDAL({
+      ancestors: 3,
+      children: 3,
+      generations: 4,
+    });
+    // save a reference to the `_id` supported by the mock data layer
+    databaseDocumentID = dataAccessLayerMetadata.databaseDocumentID;
+    // create a resolver simulator, using the data access layer and an arbitrary component instance ID
+    simulator = new Simulator({
+      databaseDocumentID,
+      dataAccessLayer,
+      resolverComponentInstanceID,
+      indices: [],
+    });
+  });
+
+  describe('when navigating to a node that is not loaded and then back to the process node list', () => {
+    beforeEach(async () => {
+      // If the camera has not moved it will return a node with ID 2kt059pl3i, this is the first node with the state
+      // loading that is outside of the initial loaded view
+      const getLoadingNodeInList = async () => {
+        return (
+          await simulator.resolveWrapper(() =>
+            simulator.testSubject('resolver:node-list:node-link')
+          )
+        )
+          ?.findWhere((wrapper) => wrapper.prop('data-test-node-state') === 'loading')
+          ?.first();
+      };
+
+      const loadingNode = await getLoadingNodeInList();
+
+      if (!loadingNode) {
+        throw new Error("Unable to find a node without it's node data");
+      }
+      loadingNode.simulate('click', { button: 0 });
+      // the time here is equivalent to the animation duration in the camera reducer
+      simulator.runAnimationFramesTimeFromNow(1000);
+
+      await simulator.resolveWrapper(() => simulator.selectedProcessNode('2kt059pl3i'));
+    });
+
+    it('should mark the node as terminated in the node list', async () => {
+      const getNodeInPanelList = async () => {
+        return (
+          await simulator.resolveWrapper(() =>
+            simulator.testSubject('resolver:node-list:node-link')
+          )
+        )
+          ?.findWhere((wrapper) => wrapper.prop('data-test-node-id') === '2kt059pl3i')
```

Comment:
This one is tough. What this test is trying to check for is that a particular node in the process list of the panel changes from loading to terminated. Unfortunately with out adding the prop we don't have a way of finding a specific node in the list. The process name wouldn't help us here because we can have multiple processes with the same name.